catalogue/webapp/pages/search/index.tsx

```diff
@@ -90,6 +90,20 @@ const SectionTitle = ({ sectionName }: { sectionName: string }) => {
   );
 };
 
+const StoryPromoContainer = styled.div.attrs({
+  className: 'container container--scroll',
```

Comment:
I've just merged a [PR that removed the container class](https://github.com/wellcomecollection/wellcomecollection.org/pull/10019), you'll have to replace it 😬